src/lxqtsystemtrayicon.cpp

```diff
@@ -355,7 +355,7 @@ QPlatformMenu *LXQtSystemTrayIcon::createMenu() const
 QRect LXQtSystemTrayIcon::geometry() const
 {
     // StatusNotifierItem doesn't provide the geometry
-    return QRect();
+    return {};
```

Comment:
This is new to me. Just asking, not judging: Why not returning a `QRect`?

Reply:
Note: I know what ` return {};` means but `return QRect();` seems completely OK to me.

Reply:
The explicit call to the constructor was replaced by a braced initializer list.
We don't need to duplicate (writing) the return type in the function definition and the return statement.

Reply:
> Note: I know what ` return {};` means but `return QRect();` seems completely OK to me.

That's more a question of style than anything else. Easier to write.

Reply:
I know that (as I said above) but I thought you only wanted to replace deprecated usages is these PRs (especially `NULL`). This one was an exception -- I think it's only a matter of taste.

Reply:
Anyhow, I have nothing against these things. Different coders have different styles.

Reply:
Yes, it can be viewed as a matter of taste. More or less as `auto`.
It can be drooped. .

Reply:
I completely agree.

Personally, I like `auto` because it can be used multiple times inside a code block. As for returning, I prefer explicitness but, in the end, it makes no difference.

So, never mind... just a chat.

Reply:
I'm Ok with that commit not being merged.
It's not relevant.

Reply:
No, I didn't mean that; I was just curious. It's good to have once instance of it somewhere.